site/docs/patterns/keyboard-shortcuts.mdx

```diff
@@ -0,0 +1,220 @@
+---
+title: Keyboard Shortcuts
+description: "Keyboard shortcuts allow users to execute actions, using a keyboard, that are usually performed with a mouse or track pad. They increase the efficiency and productivity of users who primarily rely on their keyboard when interacting with applications."
+layout: DetailPattern
+tags:
+  - pattern
+aliases:
+  - /salt/patterns/keyboard-shortcuts
+data:
+  resources:
+    [
+      { href: "https://storybook.saltdesignsystem.com/?path=/story/patterns-keyboard-shortcuts--with-dialog", label: "Examples" },
+    ]
+  components: ["Avatar", "Button", "Flow layout", "Icon", "Stack layout", "Text", "Tooltip"]
+---
+The keyboard shortcuts displays a comprehensive, searchable list of all available shortcuts, helping users quickly learn and reference key combinations to navigate and perform actions efficiently.
+
+ <Diagram
+    src="/img/patterns/keyboard-shortcuts/keyboard-shortcuts.png"
+    alt="Keyboard Shortcuts"
+  />
+
+## When to use
+
+Use this pattern to:
+- Increase task efficiency by enabling users to perform actions quickly.
+- Provide a true alternative to mouse or track pad navigation, supporting users who prefer or require keyboard input.
+- Clearly communicate available shortcuts to help users discover and learn efficient workflows.
+- Allow users to search for and execute actions directly from the keyboard, improving accessibility and user experience.
```

Comment:
this does not sound right... you can search and you can execute but you do not execute an action from the search. Maybe search needs a clearer description ?